pom.xml

```diff
@@ -49,6 +49,7 @@
         <license>COPYING</license>
         <netbeans.compile.on.save>none</netbeans.compile.on.save>
         <nashorn.version>15.4</nashorn.version>
+        <graaljs.version>24.1.1</graaljs.version>
```

Comment:
Graal.js 24.1.1 - if it can run on JDK21 and JDK25, then good for now. Btw. version 25 should already be out as well.